packages/api/README.md

```diff
@@ -58,6 +58,7 @@ Reference:
 * [Documents by Data Contract](#documents-by-data-contract)
 * [Identity by Identifier](#identity-by-identifier)
 * [Identity by DPNS](#identity-by-dpns)
+* [Identity Withdrawals](#identity-withdrawals)
```

Comment:
Apply documentation for decodeStateTransition

Reply:
Also mention null result set for identities withdrawals

Reply:
✅